### ISSUE TITLE
Any flatpack package or aur way to install?

### ISSUE THREAD

**Issue by gnacho** [fabiocolacio/Marker#40]
This program looks like everything I wanted and needed, but I'd love to have an easy way to install it. Any possibility to have a flatpack package or be in AUR (Arch) repository?

**Comment by fabiocolacio** [comment]
As of right now, there are no packages, but I plan to add them in the coming weeks. I hope to add them very soon, but i am not familiar with the process so there might be some delays. Sorry about that!

**Comment by gnacho** [comment]
Many thanks for the response. I posted already in AUR the request:

https://bbs.archlinux.org/viewtopic.php?pid=1737174#p1737174

**Comment by mmetak** [comment]
https://aur.archlinux.org/packages/marker-git/

**Comment by Mandarancio** [comment]
As suggested by @bertob in #110  it would be interesting to distribute it via [flathub](https://flathub.org/)

**Comment by jsparber** [comment]
This are the flathub [guidelines](https://github.com/flathub/flathub/wiki/Package-Guidelines)
I think we are ready for flathub. \o/
We only need to create apull request against new-pr branch in the [flathub/flathub](https://github.com/flathub/flathub)

If you are ok with it I can make the PR? It shouldn't make any problems with ownership.

**Comment by fabiocolacio** [comment]
Does the flathpak build succeed at the head of the master branch currently? If so, I will make a new release for ``2018.01.24``.

I have no qualms with you making the PR as long as there are no problems with ownership. You are much more familiar with the format, so you will do a better job of solving any issues that arise.

Thank you both very much for helping with this @jsparber and @bertob!

**Comment by jsparber** [comment]
Yeah, the master branch is building. Ok will do the PR tomorrow, if you create the release.

**Comment by fabiocolacio** [comment]
Alright the release has been made!

**Comment by jsparber** [comment]
Made a pull request. I think you should get commit rights, @fabiocolacio since you are the maintainer. For new releases you have just to change the manifest in the repo they are creating for Marker, adding the new tag. Also you would have to change the app data file for the release history.

**Comment by fabiocolacio** [comment]
@jsparber Good work getting the PR merged, but I still don't see Marker on flathub. Is there more that needs to be done on our end, or is it just a matter of time?

**Comment by TingPing** [comment]
https://beta.flathub.org/apps/details/com.github.fabiocolacio.marker

Will have to investigate why the icon isn't showing up.

**Comment by fabiocolacio** [comment]
That's fantastic! If I need to change something on my end to fix the icon let me know.

Thanks for @TingPing and everyone else at Flathub for providing such an excellent service!

**Comment by jsparber** [comment]
@TingPing From where does flathub get the icon, because gnome shows the icon as expected?

**Comment by jsparber** [comment]
@TingPing I have no idea if i'm on the right track, it could be that Marker doesn't export 128px and 64px icons, actually no icons expect the svg.

**Comment by TingPing** [comment]
If you don't have the app installed it should be the same method between gnome-software and the website.

You should add a 64/128 icon though. This is a simple way to do it: https://github.com/flathub/org.filezillaproject.Filezilla/commit/75f9d206a0a4f0ad8817e570d8f7fbad7ca5785d

**Comment by jsparber** [comment]
I added the code to the manifest for exporting the icons as 64px and 128px.
https://github.com/flathub/com.github.fabiocolacio.marker/commit/b10bf3680e8acbee569a16b53869464e738bc8b5

@TingPing  When/How is the website updated?

**Comment by TingPing** [comment]
The old site is generated by hand, which is why it is outdated: https://flathub.org/apps.html

Next time I get a chance I'll talk with the dev of the new site to figure how why some icons aren't showing up still.

**Comment by jmmourinho** [comment]
I don't know if it is just a Linux Mint thing, but after just installing 18.3 which comes with flatpak and flathub integration in the software store, Marker is one of the few apps without a description.

![Screenshot from Linux Mint 18.3](https://user-images.githubusercontent.com/7329636/35783325-1f23a2ac-09fd-11e8-9912-2493524741f5.png)

On the other hand on [Flakhub](https://flathub.org/apps/) the description is there but the image is missing...

**Comment by TingPing** [comment]
@jmmourinho Yes the missing image was a Flatpak bug that has been fixed. Icons will either be fixed on next Flatpak release or worked around in Flathub.

**Comment by tim77** [comment]
📦 **Marker** RPM Fedora package https://github.com/fabiocolacio/Marker/issues/255

**Comment by Mandarancio** [comment]
Hi I made a new release, someone could help to publish it on flathub?
Thanks! :)

**Comment by gasinvein** [comment]
@Mandarancio Since you are a collaborator, you should probably ask @TingPing or other Flathub admins to give you push access to https://github.com/flathub/com.github.fabiocolacio.marker, so you'll be able to publish new releases yourself.
